ca_cdk_constructs/edge_services/waf_rule_templates.py

```diff
@@ -19,6 +21,9 @@ def managed_rule_group_property(
     count_only: bool = False,
     rules_to_exclude: list[str] = [],
     cloud_watch_metrics_enabled: bool = False,
+    managed_rule_group_configs: IResolvable
+    | Sequence[IResolvable | waf.CfnWebACL.ManagedRuleGroupConfigProperty | Dict[str, Any]]
```

Comment:
for the types here they might seem weird but I simply copied the upstream ones (the ones from aws-cdk-lib)